wiki/Performance_points/id.md

```diff
@@ -0,0 +1,132 @@
+
+*Catatan: Sebuah jumlah kecil bonus pp diberikan berdasarkan pada skor yang kamu punya pada jumlah map Ranked*
+
+### Sistem pembobotan
+
+Sistem pemboboton nya adalah rumus yang sederhana digunakan setelah perhitungan penuh dari berapa poin perfoma sebuah play layak. Rumus ini digunakan untuk mengurangi jumlah pp yang diperoleh berdasarkan penempatan permainan tersebut di skor terbaik pemain.
+`Total pp = p * 0.95^(n-1)`<!-- may want a graphic representation here? (instead of code block) -->
+
+Mengenai rumus yang ada diatas, *p* menggambarkan setiap masing-masing jumlah penuh pp dari skor (pra-pembobotan), dan *n* adalah penempatan di ranking performa terbaik pemain. Sebagai contoh, jika skor terbaik permain adalah 110pp, 100pp, 100pp, 90pp, dan 80pp, lalu skor yang telah dibobotkan akan menjadi sekitar 110pp, 95pp, 90pp, 77pp, dan 65pp.
+
+### Pembidikan
+
+*Pembidikan* adalah nilai inti yang mempertimbangkan betapa sulitnya mengenai not secara berurutan dan konsisten dalam sebuah beatmap.
+
+Elemen seperti [approach rate](/wiki/Beatmapping/Approach_rate) dan [mods](/wiki/Game_modifier) tertentu (yaitu [Flashlight](/wiki/Game_modifier/Flashlight), [Hidden](/wiki/Game_modifier/Hidden) dan [Hard Rock](/wiki/Game_modifier/Hard_Rock)) membuat menavigasi kursor dengan cepat dan akurat jauh lebih sulit, dan dengan demikian memengaruhi jumlah pp yang diberikan oleh skor tersebut.
+
+Dalam kasus seperti [osu!standard](/wiki/Game_mode/osu!), beatmap dengan [jumps](/wiki/Beatmap/Pattern/Jump) yang sangat besar itu dipertimbangkan sebagai beatmaps "bidikan tinggi", dan oleh karena itu, sering memberikan skor pp yang sangat tinggi. Demikian juga, beatmaps dengan lebih banyak hyperdashing di [osu!catch](/wiki/Game_mode/osu!catch) itu akan dipertimbangkan dengan sama. Pidikan itu tidak dipertimbangkan didalam gamemode seperti [osu!taiko](/wiki/Game_mode/osu!taiko) dan [osu!mania](/wiki/Game_mode/osu!mania).
+
+### Kecepatan
+
+*Kecepatan* adalah nilai inti yang mempertimbangkan pada nilai mana sebuah menyajikan elemen untuk play tersebut.
+
+Beatmap dengan jumlah hit objek yang tinggi dalam waktu yang singkat juga dipertimbangkan untuk mempunyai jumlah kecepatan yang tinggi. Dalam aspek spesifik ini, semakin cepat kecepatan beatmap itu, semakin sulit beatmap tersebut itu menjadi, karena itu, memberikan jumlah pp yang diperoleh lebih besar.
+
+Sebagai hasil dari ini, mod seperti [Double Time](/wiki/Game_modifier/Double_Time) dan [Half Time](/wiki/Game_modifier/Half_Time) secara signifikan mempengaruhi kecepatan beatmap yang dipertimbangkan oleh algoritma poin performa. Demikian juga, mod-mod itu juga secara signifikan mempengaruhi jumlah pp yang diperoleh saat digunakan.
+
+### Akurasi
+
+*Lihat juga: [Akurasi](/wiki/Accuracy)*
```

Comment:
related to http://github.com/ppy/osu-wiki/pull/5119#issuecomment-806474761 :

`` ### Accuracy

*Accuracy* merupakan variabel perhitungan pp yang menyangkut seberapa akurat seorang pemain dalam mengenai [*hit object-hit object*](/wiki/Hit_object) yang ada pada waktunya...``

Reply:
not sure if ```Akurasi``` should be renamed to ```Accuracy```, since it makes alot of sense to just use that one, as people would likely to say "Akurasi" than "Accuracy" for this one

to extend further, the ID version of Accuracy wiki also uses "Akurasi"
https://osu.ppy.sh/wiki/id/Accuracy

Reply:
The use of `akurasi` won't confuse anyone (Indonesians). Afaik, there are many players say `akurasi` instead of `accuracy` in #indonesian, that I think using `akurasi` isn't a big problem here. Also, agree with wowcake's statement about the ID ver of Accuracy wiki.

Exceptions, if we want to use `accuracy` in the future, it's good to change `akurasi` to `accuracy` here